http3/http_stream.go

```diff
@@ -147,10 +148,12 @@ type requestStream struct {
 	reqDone            chan<- struct{}
 	disableCompression bool
 	response           *http.Response
+	trace              *httptrace.ClientTrace
 
 	sentRequest   bool
 	requestedGzip bool
 	isConnect     bool
+	firstByte     bool
```

Comment:
I believe you don't need this anymore, since this is now tracked by the wrapper reader, right?

Reply:
It still needs to be retained temporarily because `ReadResponse` will be called multiple times, and each call will new `tracingReader`. Therefore, this information needs to be preserved in the `requestStream`.